upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -95,7 +97,50 @@ func (b *BootstrapChannelBuilder) Build(c *fi.ModelBuilderContext) error {
 			Location:  fi.String(manifestPath),
 			Name:      fi.String(name),
 		}
+	}
+
+	if b.ClusterAddons != nil {
+		key := "cluster-addons.kops.k8s.io"
+		version := "0.0.0"
+		location := key + "/default.yaml"
+
+		a := &channelsapi.AddonSpec{
+			Name:     fi.String(key),
+			Version:  fi.String(version),
+			Selector: map[string]string{"k8s-addon": key},
+			Manifest: fi.String(location),
+		}
+
+		if a.Id != "" {
```

Comment:
How can this ever be true?

Reply:
Good call - removed (I think I was aiming for parity with the existing code, but I don't know)